scripts/PauseAtHeight.py

```diff
@@ -42,6 +42,14 @@ def getSettingDataString(self):
                     "unit": "mm",
                     "type": "float",
                     "default_value": 0
+                },
+                "extrude_ammount":
```

Comment:
Ah, right, didn't see that.
Fixing both would be nice, though it's not your obligation.